### DISCUSSION TITLE
How can I turn off the redirecting to a educational page whenever I launch the site

### DISCUSSION THREAD

**Discussion by hsafha** [UseInterstellar/Interstellar#341] [General]
My school has goguardian and I need to launch the site within another app so my screen can't be seen. However when I open the site on this app I get redirected to classroom or nasa.

**Comment by Voxxie2k** [comment]
if it is on chrome, which it most likely is, go to chrome://settings and search up site settings. scroll ALL the way to the bottom to where it says pop ups and redirects. in the top right, enter the URL, and turn off redirects
